### ISSUE TITLE
Website API returns curl URLs "http" and not "https"

### ISSUE THREAD

**Issue by theblaggy** [TheMorpheus407/BattleOfAI#4]
The URLs returned by the website contain only "http" so you will get a 301 Moved Permanently

**Comment by TheMorpheus407** [comment]
Can you give an example URL?
The internal links only reference the file.
Do you mean the APIs?
They indeed only use http, however it should work anyways.

**Comment by Vitroxyn** [comment]
@the-blaggyS Do you probably meant the documentation of the APIs ([Games](https://games.battleofai.net/api/) & [Iam](https://iam.battleofai.net/api/)), their examples produces a curl command e.g.
`curl -X POST "http://iam.battleofai.net/api/iam/login" -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"username\": \"string\",  \"password\": \"string\"}"`
which is invalid because it has the http protocol?

**Comment by TheMorpheus407** [comment]
Yeah, that's the representation of swagger, cant change that sadly. Executing it online should work however?

**Comment by theblaggy** [comment]
Yes, online it´s working fine, but not in terminal.

**Comment by felbinger** [comment]
@TheMorpheus407 We could change that...
```python
from werkzeug.contrib.fixers import ProxyFix
app.wsgi_app = ProxyFix(app.wsgi_app)
```
